### PR TITLE
MLOB-1014: Update LLM Observability API page

### PR DESCRIPTION
**Pull request by alexbarksdale**
### What does this PR do? What is the motivation?

This pr updates the API page:
- Removes the beta banner at the top
- Update endpoints for span and eval-metric submission
- Update the eval-metric api request body

### Merge instructions

- [x] Please merge after reviewing

### CODE REVIEW

**Review by rtrieu — APPROVED** [review]
LGTM

File: content/en/llm_observability/api.md [hunk 1]
```diff
@@ -259,14 +257,16 @@ Evaluations require a `span_id` and `trace_id`.
...
+          "ml_app": "weather-bot",
+          "timestamp_ms": 1609459200,
           "metric_type": "categorical",
           "label": "Sentiment",
           "categorical_value": "Positive"
         },
         {
           "span_id": "20245611112024561111",
           "trace_id": "13932955089405749200",
+          "ml_app": "weather-bot",
```
Comment (gary-huang) [inline content/en/llm_observability/api.md:269]: does this need a timestamp_ms as well?

Comment (gary-huang) [reply content/en/llm_observability/api.md:269]: or are we trying to show that timestamp is optional? i feel like we should push / encourage submission of timestamp regardless

Comment (alexbarksdale) [reply content/en/llm_observability/api.md:269]: nah, it was a miss. timestamp is required, ty

File: content/en/llm_observability/api.md [hunk 2]
```diff
@@ -310,6 +311,7 @@ Evaluations require a `span_id` and `trace_id`.
           "id": "cdfc4fc7-e2f6-4149-9c35-edc4bbf7b525",
           "span_id": "20245611112024561111",
           "trace_id": "13932955089405749200",
+          "ml_app": "weather-bot",
```
Comment (gary-huang) [inline content/en/llm_observability/api.md:314]: do we need a timestamp here too?

**Review by gary-huang — APPROVED** [review]

**Comment by gary-huang** [comment]
guessing we merge this when @barieom or Shri is ready to send the announcement? (since it removes the beta banner?)

**Comment by alexbarksdale** [comment]
> guessing we merge this when @barieom or Shri is ready to send the announcement? (since it removes the beta banner?)

yeah, got an okay from barry to merge

**Pull request merged**